Tests/installation_tests/cocoapods/with_frameworks/CocoapodsTestTests/CocoapodsTestTests.swift

```diff
@@ -28,7 +28,7 @@ class CocoapodsTestTests: XCTestCase {
     
     func testPerformanceExample() {
```

Comment:
Yeah all of the integration test projects have these empty default test stuff in them (this one is just in the diff because the swift3 migrator made changes). I think we might want to go through and either delete them all or add tests to them all, but I think that is a separate PR?